# timestamp information
uint64 framenumber
time stamp # start of image frame acquisition timestamp
# camera information
string cam_id

TrackedObject[] tracked_objects
